### ISSUE TITLE
Comments for blank imports in multiple ImportSpec

### ISSUE THREAD

**Issue by james-d-elliott** [incu6us/goimports-reviser#1]
Hi there love the tool! Works pretty much exactly as expected.

golint expects blank imports to have a comment justifying them which have to exist directly above the import. Currently goimports-reviser moves those to the wrong location. They should remain above the import (seems like into a line of their own).

Edit: My mistake, they can exist directly after them on the same line as well. Still an issue but this may be easier to solve!

**Comment by incu6us** [comment]
Hi,
Thank for the issue! For now, this feature doesn't implement, because it is a very rare option to use comments for imports. I will check it when will have time. Probably, you could take a look into it? Will be glad to take some help :)

**Comment by james-d-elliott** [comment]
Understandable. Yes, I can take a look sometime too, if I figure it out I'll PR. Thanks again for the amazing tool. :)

**Comment by incu6us** [comment]
@james-d-elliott For now, I've removed all the comments for import spec([v0.0.6](https://github.com/incu6us/goimports-reviser/tree/v0.0.6)). Because there is an interesting situation in case:
```
import (
         // context comments here
	"context"

        // some other comments

        // time package comment
	"time"

       // possible line of comments
)
```
So, the question is - what will should I do with this hell? It will break the main idea of the import's groups. The aim of the tool is to make code cleaner and for imports I see only one way - it skips all of the comments for this block.

**Comment by james-d-elliott** [comment]
Makes sense, that's really hard to manage. golint allows for the blank import to have the comment on the same line as the import. For example:

```go
import (
	"database/sql"
	_ "github.com/jackc/pqx" // comment to satisfy golint
)
```

So maybe this is easier to handle? I had a look but wasn't sure on a solution from your code.

**Comment by incu6us** [comment]
fixed with the same line([v0.0.7](https://github.com/incu6us/goimports-reviser/tree/v0.0.7))

**Comment by james-d-elliott** [comment]
Oh well done, works flawlessly!
